presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotPushdownUtils.java

```diff
@@ -48,6 +51,10 @@
 
 public class PinotPushdownUtils
 {
+    public static final String DISTINCT_COUNT_FUNCTION_NAME = "distinctCount";
```

Comment:
where do we have this function?

Reply:
This function is defined in Pinot , please refer to [here](https://docs.pinot.apache.org/pinot-user-guide/pinot-query-language/how-to-handle-unique-counting#accurate-results)

Reply:
Let's put `PINOT_` for these consts to denote they are Pinot functions. Also break an empty between public and private consts.

Reply:
Done. Thanks!